### DISCUSSION TITLE
Добавить поле картинки для товара (поля тип Файл заблокированы на Demo-сервере)

### DISCUSSION THREAD

**Discussion by tormozit** [totumonline/totum-ru-issues-and-discussions#194] [Как это сделать?] [answered]
Тип "Картинка" у поля не нашел. Поэтому выбрал тип "Файл". Но загрузить файл даже 300 байт не дает. 
![изображение](https://user-images.githubusercontent.com/11896251/184503684-763e4bd5-0c13-412b-9152-0645d4bb0531.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
### Если это на Демо-сервере — то там заблокировано добавление файлов.

**Comment by tormozit** [comment]
Да. На демо сервере. Было бы как минимум гуманно писать вместо "Ошибка" что то типа "запрещено на демо сервере".
По сути то есть что ответить? Организовать вывод картинки в списке или хотя бы в окне объекта можно? Может в каком то обучающем видео показано?

**Reply by ttm-alexeypolunin** [reply]
Угу, надо написать — эта задача где-то есть даже в списках.

**https://ru.docs.totum.online/fields#type-file**

<img width="1056" alt="Снимок экрана 2022-08-13 в 22 27 52" src="https://user-images.githubusercontent.com/55755565/184508115-1378d329-8482-491d-8158-7466483a8550.png">

<img width="1062" alt="Снимок экрана 2022-08-13 в 22 28 06" src="https://user-images.githubusercontent.com/55755565/184508124-ebff4ca7-ea5f-4415-80b0-8fe10807ae50.png">

<img width="1048" alt="Снимок экрана 2022-08-13 в 22 29 40" src="https://user-images.githubusercontent.com/55755565/184508150-19eadf29-1775-4cd6-8d8a-74b4919d7be0.png">

<img width="1533" alt="Снимок экрана 2022-08-13 в 22 30 00" src="https://user-images.githubusercontent.com/55755565/184508158-7016ea25-3f29-4e5a-8e77-1ec47391f7c8.png">
